e2e/services/browser.ts

```diff
@@ -96,6 +96,12 @@ export class TestBrowserService {
 
   static async closeRemoteBrowser(connectParams?: ConnectRetryParams) {
     const browser = await TestBrowserService.getRemoteBrowser(connectParams);
+    const ctxs = await browser.contexts();
+    for (const ctx of ctxs) {
+      await ctx.close();
+    }
     await browser.close();
```

Comment:
closing browser, closes the underlying context. Am i missing something?

Reply:
@devcorpio and I tried running tests with closing browser only, and closing contexts first then closing the browser(we ran 5 consecutive test runs for both cases). Closing contexts first were more reliable than the other.

Reply:
That seems like a bug to me in PW. I have seen in their code that they dont close the context as well.